src/redux/api/book-club/book-club-membership.api.slice.ts

```diff
@@ -79,6 +83,15 @@ const bookClubMembershipAPISlice = api.injectEndpoints({
         url: `${props.API_PATHS.MEMBERSHIPS}/${membership.bookClub.name}/${membership.reader.id}`,
         method: 'DELETE'
       })
+    }),
+
+    // Mutation for changing the owner of a book club
```

Comment:
Rename to makeOwner; This doesn't change or swap ownership, this just makes another reader an owner.